### PR TITLE
Updated With Enter as send Key Feature

### PR DESCRIPTION
**Pull request by sagarchoudhary96**
Issue 48 Fixed. Added Enter Key as Send Feature. User can select between
to choose whether to use Enter key to send message or  function as
normal .

### CODE REVIEW

**Comment by sagarchoudhary96** [comment]
@sudheesh001 i have made changes as per Code Practices

File: app/src/main/res/xml/pref_settings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    android:title="@string/settings_title">
+
+    <SwitchPreference
+        android:title="@string/settings_enterPreference_label"
+        android:summaryOff="@string/settings_enterPreference_summaryOff"
+        android:summaryOn="@string/settings_enterPreference_summaryOn"
+        android:id="@+id/enter_key_pref"
+        android:defaultValue="false"
+        android:key="@string/settings_enterPreference_key"/>
+</PreferenceScreen>
```
Comment (daminisatya) [inline app/src/main/res/xml/pref_settings.xml]: Add a new line after last line.

**Comment by sagarchoudhary96** [comment]
@sudheesh001 can u let me know the changes needed?

**Comment by sagarchoudhary96** [comment]
@daminisatya need to add newline  after line number 13?

**Comment by daminisatya** [comment]
@sagarchoudhary96 yes

**Comment by sagarchoudhary96** [comment]
@daminisatya okay just a min :)

**Pull request closed without merging**